libr/cons/cons.c

```diff
@@ -730,10 +731,18 @@ R_API void r_cons_flush(void) {
 
 	if (I.is_interactive && I.fdout == 1) {
 		/* Use a pager if the output doesn't fit on the terminal window. */
-		if (I.pager && *I.pager && I.context->buffer_len > 0 && r_str_char_count (I.context->buffer, '\n') >= I.rows) {
+		if (pageable && I.pager && *I.pager && I.context->buffer_len > 0 && r_str_char_count (I.context->buffer, '\n') >= I.rows) {
 			I.context->buffer[I.context->buffer_len - 1] = 0;
-			r_sys_cmd_str_full (I.pager, I.context->buffer, NULL, NULL, NULL);
-			r_cons_reset ();
+			if (*I.pager == '.') {
```

Comment:
I'd use `strcmp` to avoid considering `scr.pager=./mybin` as internal less.

Reply:
i think . is a pretty bad name, in fact i was thikning in using .. like in ~.. but wanted to discuss without overengineering. also scr.pager=? doesnt shows anything

Reply:
Ok, but in any case it makes sense to use `strcmp`. The compiler is anyway smart enough to reduce it to just a 2bytes check i think.